op-service/txmgr/txmgr_test.go

```diff
@@ -772,7 +772,6 @@ func TestIncreaseGasPriceEnforcesMinBumpForBothOnTipIncrease(t *testing.T) {
 // but the tip doesn't, both values are increased by 10%
 // TODO(CLI-3620): This test will fail until we implemented CLI-3620.
 func TestIncreaseGasPriceEnforcesMinBumpForBothOnBaseFeeIncrease(t *testing.T) {
```

Comment:
Yea, I was thinking about refactoring the gas price test. Particularly that it is a deterministic function, it will be way easier to test b/c it doesn't need all the test setup.

Reply:
The `doGasPriceIncrease` function is really nice. I also added a second set of table tests just for the `updatesFees` function.